presto-main/src/main/java/io/prestosql/execution/MemoryRevokingScheduler.java

```diff
@@ -245,7 +245,7 @@ private void requestRevoking(MemoryPool memoryPool, Collection<SqlTask> sqlTasks
     {
         AtomicLong remainingBytesToRevokeAtomic = new AtomicLong(remainingBytesToRevoke);
         sqlTasks.stream()
-                .filter(task -> task.getTaskStatus().getState() == TaskState.RUNNING)
+                .filter(task -> task.getTaskStatus().getState().isRunning())
```

Comment:
We shouldn't request revoking from `FLUSHING` task as no drivers are running which could revoke memory